docs/learn/learn-accounts.md

```diff
@@ -446,7 +446,7 @@ Currently, the **DepositBase** equals `deposit(1, 88)` (key size is 32; value is
 bytes) and the **DepositFactor** equals `deposit(0, 32)` (additional address of 32 bytes).
 
 The deposit function in JavaScript is defined below, cribbed from the
-[Rust source](https://github.com/paritytech/polkadot/blob/master/runtime/polkadot/src/constants.rs#L26).
+[Rust source](https://github.com/paritytech/polkadot/blob/master/runtime/polkadot/constants/src/lib.rs#L30).
```

Comment:
as the lines change, better to omit
```suggestion
[Rust source](https://github.com/paritytech/polkadot/blob/master/runtime/polkadot/constants/src/lib.rs).
```

Reply:
agreed, I was updating what was existing but I guess we can just point to the `lib.rs` file here?

Reply:
Yes, the context is important, actually evaluate if it makes sense to only reference the file, you may need to add " look for X in this source" to help hint at where it lives